clients/admin-ui/src/features/common/Icon/database/DatabaseIcon.tsx

```diff
@@ -0,0 +1,12 @@
+import { createIcon } from "fidesui";
```

Comment:
Copied the icons from the D&D branch but I placed them in a more generic folder so we can use them for both features.